modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc

```diff
@@ -0,0 +1,105 @@
+
+= Creating Client Applications with Connected Apps
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: grant access, api, contracts
+
+
+For automation use cases, you can use Connected Apps to create and manage client applications.
+
+Connected Apps help user delegate their access without sharing sensitive credentials or giving full control of their applications to third parties. Actions taken by connected apps are audited, and users can revoke access at any time.
+
+
+== Creating a Connected App
+
+To create a Connected App, see xref:access-management::connected-apps-developers.adoc[].
+
+When you create the Connected App to manage client applications, ensure that you:
+
+. Select *App acts on its own behalf (client credentials)* for *Type*.
+. Add the *Application Creator* scope.
+
+image::connected-app-client-apps.png[Screenshot calling out the location of App acts on its own behalf (client credentials) and Application Creator]
+
+[calloutlist]
+. *App acts on its own behalf (client credentials)* type
+. *Add Scopes* button to add *Application Creator* scope
+
+The new Connected App can now create client applications and manage the client applications that it creates.
+
```

Comment:
Since this is doc and not release notes, we should not say "can now create". The doc always document the current state of the product. If you must talk about differences from the past in doc vs. RN, these should be in a topic specifically for that purposes and/or have specific release info on when the change was introduced.